.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "airbnb",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+    }
+}
```

Comment:
Please add EOF here

Reply:
Thank you, it is done.

Reply:
Please add `"esversion": 6,` in this file for the hound to sniff the version of ECMAScript we are using

Reply:
I believe it's covered by this line `"ecmaVersion": 2018`.  Also, let me point out we are using `ESLint` not `JSHint`